Mathlib/MeasureTheory/Measure/WithDensityVectorMeasure.lean

```diff
@@ -129,6 +129,23 @@ theorem withDensityᵥ_smul' {𝕜 : Type*} [NontriviallyNormedField 𝕜] [Norm
   withDensityᵥ_smul f r
 #align measure_theory.with_densityᵥ_smul' MeasureTheory.withDensityᵥ_smul'
 
+theorem withDensityᵥ_smul_eq_withDensityᵥ_withDensity {f : α → ℝ≥0} {g : α → E}
+    (hf : Measurable f) (hfg : Integrable (f • g) μ) :
```

Comment:
Can you relax the measurability condition to ae measurable

Reply:
For this and the other theorems, should I create a two theorems, one for measurable and one for ae measurable, or should I only keep the ae measurable theorem?

Reply:
I would only keep the ae measurable theorems as it is easy to get ae measurable from measurable

Reply:
Great, that is what I did in the new commits